app/src/main/java/com/kms/cura/model/Settings.java

```diff
@@ -0,0 +1,8 @@
+package com.kms.cura.model;
+
+/**
+ * Created by linhtnvo on 6/6/2016.
+ */
+public class Settings {
+    public static final String URL_CREATE_PATIENT_API = "http://10.0.3.2:8080/com.kms.cura_server-0.1/cura_server/user/createPatient";
```

Comment:
should separate URL_CREATE_PATIENT_API into SERVER_URL and CREATE_PATIENT_API for re-use